webapp/components/_new/generic/BaseButton/BaseButton.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <button
+    :class="buttonClass"
+    :disabled="loading"
+    :type="type"
+    @click.capture="event => $emit('click', event)"
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-14T17:54:11Z" title="Tuesday, January 14th 2020, 6:54:11 pm +01:00">Jan 14, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,144 @@
+<template>
+  <button
+    :class="buttonClass"
+    :disabled="loading"
+    :type="type"
+    @click.capture="event => $emit('click', event)"
```


maybe it has to do with this? :point_up_2:

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-15T07:54:46Z" title="Wednesday, January 15th 2020, 8:54:46 am +01:00">Jan 15, 2020</time>_

---


yes... somehow. see above! :grin: